apm-agent-core/src/main/java/co/elastic/apm/agent/impl/transaction/TraceContext.java

```diff
@@ -242,6 +242,12 @@ public boolean asChildOf(TraceContext child, Object ignore) {
     @Nullable
     private String serviceName;
 
+    @Nullable
+    private String frameworkName;
```

Comment:
Better store `frameworkName` and `frameworkVersion`  in `TransactionContext`, as it's only relevant to transactions and doesn't increase the memory footprint for spans.

Reply:
fixed